.pipelines/cni/pipeline.yaml

```diff
@@ -418,6 +549,16 @@ stages:
       - windows_overlay_HNS
       - windows19_overlay_HNS
       - setup
+      - ${{if eq(parameters.upgradeScenario, true)}}:

```

Comment:
do we create clusters for the upgrade case? It seems like we skip the create stage (in cilium overlay load test template) if upgrade is true?

Reply:
yes, if we are doing the upgrade scenario then the cluster create will be skipped and the old clusters will be reused